renovate.json

```diff
@@ -27,6 +27,12 @@
       ],
       "allowedVersions": "/^[0-9]+\\.[0-9]+\\.[0-9]+$/"
     },
+    {
+      "matchPackageNames": [
+        "org.jruby:jruby"
+      ],
+      "allowedVersions": "<10.0.0.0"
```

Comment:
I'm a bit skeptical whether this comparison works in practice as "10.0.0.0" is not a semantic version, but I'm willing to just give it a try.

Reply:
See https://github.com/oss-review-toolkit/ort/issues/11167.